code/index.html

```diff
@@ -1,30 +1,107 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
+    <title>Rent-a-classic-VW</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@900&family=Raleway&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <!-- video or image as a header is cool :) went for image. Cool enough :)-->
+    <div class="hero-image">
+        <div class="hero-text">
+            <h1>RENT A CLASSIC VW</h1>
+            <p>Dreaming of those perfect insta-moments? Curious about the vanlife- or vw-community?</p>
+            <p>Rent one of our cars and make YOUR dreams come true.
+            </p>
+        </div>
+    </div>
+
+
+    <main>
+        <!-- Signup form -->
+        <section class="booking">
+            <div>
+                <h2>Booking</h2>
+            </div>
+            <form class="form" action="http://httpbin.org/anything" method="POST">
+                <div class="form-group">
+                    <label for="name">Name:</label>
+                    <input id="name" type="text" name="name" placeholder="Your full name here" name="nameValue" required>
```

Comment:
Great job, very clean code + great class naming! Nevertheless, consider these changes :
1. Your attributes could be moved to separate lines since there are so many of them (same goes for inputs below)
2. You duplicated `name` attribute over here

Reply:
I'm trying to do like you say, but when I save in VSCode it formats it like this.

Reply:
Ahh, I see - it varies between local setup of VS Code formatter. You can always install something called `prettier`, it's a formatter for HTML, CSS and JS files, but it's not mandatory

Reply:
I had prettier installed, and it madeit look like this. Uninstalled prettier, now I can make it look like this. I think th heroimage should be responsive now.